.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,78 @@
+name: Build and Test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+    
+    - name: Set up Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: '1.16'
+    
+    - name: Build kommando CLI
+      run: |
+        go build -o ./build/kommando ./cmd/kommando/...
+        cp build/kommando ./example/
+    
+    - name: Test add command
```

Comment:
Move test cases out of the workflow file, ideally we should just call one command from here to run all the test cases and get the result. Is it possible to use the built in test utility in Go to help with test cases?